examples/labelelide-example.ts

```diff
@@ -0,0 +1,308 @@
+        this._labelPass.target = this._defaultFBO;
+        this._labelPass.depthMask = true;
+
+        FontFace.fromFile('./data/opensans2048p160d16.fnt', context)
+            .then((fontFace) => {
+                for (const label of this._labelPass.labels) {
+                    label.fontFace = fontFace;
+                }
+                this._fontFace = fontFace;
+                this.updateLabels();
+                this.invalidate();
+            })
+            .catch((reason) => auxiliaries.log(auxiliaries.LogLevel.Error, reason));
+
+        this.setupScene();
+
+        return true;
+    }
+
+    /**
+     * Uninitializes Buffers, Textures, and Program.
+     */
+    protected onUninitialize(): void {
+        super.uninitialize();
+
+        this._defaultFBO.uninitialize();
+        this._labelPass.uninitialize();
+    }
+
+    /**
+     * This is invoked in order to check if rendering of a frame is required by means of implementation specific
+     * evaluation (e.g., lazy non continuous rendering). Regardless of the return value a new frame (preparation,
+     * frame, swap) might be invoked anyway, e.g., when update is forced or canvas or context properties have
+     * changed or the renderer was invalidated @see{@link invalidate}.
+     * Updates the navigaten and the AntiAliasingKernel.
+     * @returns whether to redraw
+     */
+    protected onUpdate(): boolean {
+
+        for (const label of this._labelPass.labels) {
+            if (label.altered || label.color.altered) {
+                return true;
+            }
+        }
+        return this._altered.any || this._camera.altered;
+    }
+
+    /**
+     * This is invoked in order to prepare rendering of one or more frames, regarding multi-frame rendering and
+     * camera-updates.
+     */
+    protected onPrepare(): void {
+
+        if (this._altered.canvasSize) {
+            this._camera.aspect = this._canvasSize[0] / this._canvasSize[1];
+            this._camera.viewport = this._canvasSize;
+
+            this.updateLabels();
+        }
+
+        if (this._altered.clearColor) {
+            this._defaultFBO.clearColor(this._clearColor);
+        }
+
+        this._labelPass.update();
+
+        this._altered.reset();
+        this._camera.altered = false;
+    }
+
+    /**
+     * After (1) update and (2) preparation are invoked, a frame is invoked. Renders both 2D and 3D labels.
+     * @param frameNumber - for intermediate frames in accumulation rendering
+     */
+    protected onFrame(/*frameNumber: number*/): void {
+        const gl = this._context.gl;
+
+        gl.viewport(0, 0, this._camera.viewport[0], this._camera.viewport[1]);
+
+        this._defaultFBO.clear(gl.COLOR_BUFFER_BIT | gl.DEPTH_BUFFER_BIT, false, false);
+        this._labelPass.frame();
+    }
+
+    /**
+     * After (1) update, (2) preparation, and (3) frame are invoked, a swap is invoked for multi-frame rendering.
+     */
+    protected onSwap(): void {
+    }
```

Comment:
the doc suggests that we need a swap, but this is empty. Maybe comment on why it is empty? e.g., "no accumulation in this example"?

Reply:
Same in `labeltestrenderer.ts` and `labelanchor-example.ts`?